demos/bert_question_answering/python/bert_question_answering.py

```diff
@@ -21,14 +21,47 @@
 
 import numpy as np
 
-from tokens_bert import text_to_tokens, load_vocab_file
+from tokens_bert import text_to_tokens, load_vocab_file, Token
 from html_reader import get_paragraphs
 import grpc
 import numpy as np
 from tensorflow import make_tensor_proto, make_ndarray
 from tensorflow_serving.apis import predict_pb2
 from tensorflow_serving.apis import prediction_service_pb2_grpc
 
+class ConcatenatedParagraph():
+    def __init__(self, text="", tokens=[]):
+        self.text = text
+        self.tokens = tokens
+        self.tokens_length = len(tokens)
+
+    def append_paragraph(self, text, tokens):
+        self.append_tokens(tokens)
+        self.append_text(text)
+
+    def append_text(self, text):
+        self.text += text
+
+    def append_tokens(self, tokens):
+        self.tokens += self.append_offset(tokens, len(self.text), len(self.text))
+        self.tokens_length = len(self.tokens)
+
+    def append_offset(self, tokens, s_offset, e_offset):
+        return list(map(lambda token: Token(token.id, token.start + s_offset, token.end + e_offset), tokens))
+    
+    def get_tokens_id(self):
+        return [token.id for token in self.tokens]
+
+    def get_tokens_se(self):
```

Comment:
To be honest I also don't like this "se" abberv. 
Maybe we could rename it to something like "range" or "position"?
"get_token_range" or "get_token_position" or something

Reply:
It seems like this method is not used anymore so I think just removing it should be okay.

Reply:
Even better 😆